tests/unit/stats/test_stats_exporters.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Zenodo.
+# Copyright (C) 2018 CERN.
+#
+# Zenodo is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Zenodo is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Zenodo. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this licence, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+"""Unit tests for statistics exporters."""
+from datetime import datetime, timedelta
+from mock import mock
+import pytest
+
+from flask import current_app
+from invenio_cache import current_cache
+
+from stats_helpers import create_stats_fixtures
+from zenodo.modules.stats.exporters import PiwikExporter,\
+    PiwikExportRequestError
+
+
+class MockResponse:
+    def __init__(self, json_data, status_code):
+        self.json_data = json_data
+        self.status_code = status_code
+        self.ok = status_code == 200
+
+    def json(self):
+        return self.json_data
+
+
+def mocked_requests_success(*args, **kwargs):
+    url = current_app.config['ZENODO_STATS_PIWIK_EXPORTER'].get('url', None)
+    json = {'status': 'success', 'invalid': 0}
+
+    if args[0] == url:
+        return MockResponse(json, 200)
```

Comment:
I don't think you need to check for the URL, since all of the calls inside the test will be towards the same one always.